modules/configure-acs-kube-security-platform.adoc

```diff
@@ -0,0 +1,52 @@
+[id="configure-acs-kube-security-platform"]
+= Configuring the {product-title} platform
+:context: configure-acs-kube-security-platform
+
+You can create a new integration in the {product-title} platform by using the email plugin notifier.
+
+.Procedure
+
+. Navigate to *Platform Configuration* -> *Integrations*.
+. Under the *Plugins* section, select *Email*.
+. Select the *New Integration* icon.
+. Enter a name for your email integration in the *Integration Name*.
+. For *Email Server*, enter the address of your email server.
+. Enter the `_Username_` and `_Password_` to use for authentication.
+. Optional: For *FROM*, specify the `_email FROM header_`.
+. For *SENDER*, specify the `_email SENDER header_`.
+. Enter the email address which will receive the notifications in the *Default Recipient* box or enter an annotation key for *Annotation key for recipient*.
+. Optional: Turn on the *Connect Without TLS (Unencrypted)* toggle to send email without TLS. You should not disable TLS unless you are using StartTLS.
++
+[NOTE]
+====
+Use TLS for email notifications. Without TLS, all email is sent unencrypted.
+====
+
+. Optional: To use StartTLS, select either *Login* or *Plain* for the *Use STARTTLS (Requires TLS To Be Disabled)* drop-down menu.
++
+[IMPORTANT]
+====
+With StartTLS, credentials are passed in plain text to the email server before the session encryption is established.
+
+* StartTLS with the *login* parameter sends authentication credentials in a `*base64*` encoded string.
+* StartTLS with the *plain* parameter sends authentication credentials to your mail relay in plain text.
+====
+
+To use an annotation to dynamically determine an email recipient:
```

Comment:
This flow isn't clear because its at the very bottom. Would explain the workflow more at the top.